develop-docs/self-hosted/production-enhancements/monitoring.mdx

```diff
@@ -99,6 +97,32 @@ metrics:
   prefix: "sentry.symbolicator" # Adjust this to your needs, default is "symbolicator"
 ```
 
+### Uptime Checker
+
+You can configure Uptime Checker to send metrics to Statsd server by configuring your `docker-compose.yml` file:
```

Comment:
They should be using the override file instead of the main one, right?

Reply:
I'd actually go with the second option: add these to pass in already set env variables and just document these.